registry/mainnet/interchain/squid.tokenlist.json

```diff
@@ -3595,6 +3595,38 @@
           "tokenManagerType": "mintBurn"
         }
       ]
+    },
+    "0x354931b9f1ec978dd4c474017982be103dad4faabb035d2e62fd005c9b2be9ee": {
+      "tokenId": "0x354931b9f1ec978dd4c474017982be103dad4faabb035d2e62fd005c9b2be9ee",
+      "deployer": "0xCe51D260F3c17256d18EeBaB9e5cFFC5870a3ee3",
+      "originalMinter": null,
+      "prettySymbol": "NUT",
+      "decimals": 18,
+      "originAxelarChainId": "ethereum",
+      "tokenType": "canonical",
+      "deploySalt": "0x",
```

Comment:
@NutcoinOrg kindly add the coingecko id for this token after this field. E.g`coingeckoId`: `...` and fix the { error to allow lint check to pass.